lisa/operating_system.py

```diff
@@ -551,7 +551,7 @@ class Redhat(Fedora):
 
     @classmethod
     def name_pattern(cls) -> Pattern[str]:
-        return re.compile("^rhel|Red|Scientific|acronis|Actifio$")
+        return re.compile("^rhel|Red|AlmaLinux|almalinux|Scientific|acronis|Actifio$")
```

Comment:
Add one of them is enough. Not to add unnecessary things.

Reply:
Have changed.